apps/dashboard/features/holders-and-delegates/hooks/useDelegateDelegationHistory.ts

```diff
@@ -0,0 +1,270 @@
+import { useCallback, useMemo, useState, useEffect } from "react";
+import { useGetDelegateDelegationHistoryQuery } from "@anticapture/graphql-client/hooks";
+
+// Interface for a single delegation history item
+export interface DelegationHistoryItem {
+  timestamp: string;
+  transactionHash: string;
+  delta: string; // Amount the selected address won or lost in votingPower
+  delegation: {
+    delegatorAccountId: string;
+    delegatedValue: string;
+    previousDelegate: string | null;
+    delegateAccountId: string;
+  } | null;
+  transfer: {
+    amount: string;
+    fromAccountId: string;
+    toAccountId: string;
+  } | null;
+  votingPower: string;
+  type: "delegation" | "transfer";
+  action: string;
+  isGain: boolean; // true if delegate gains voting power, false if loses
+}
```

Comment:
couldn't this be replaced by the [generated type](https://github.com/blockful/anticapture/pull/1056/files#diff-bc0b018eda2fc1dad0799640bc5ef43a5ab65ecddf943c45ea77f364ff22b9f3R2479)?